src/mcp/playwright/__tests__/markdown-extractor.password.test.ts

```
@@ -0,0 +1,76 @@
+  '<button type="submit">Continue</button>' +
+  '</form>';
+
+function fillForm(): void {
+  document.body.innerHTML = LOGIN_PAGE;
+  (document.getElementById('u') as HTMLInputElement).value = 'alice@example.com';
+  (document.getElementById('p') as HTMLInputElement).value = 'hunter2SECRET';
+  (document.getElementById('p2') as HTMLInputElement).value = 'newpassSECRET';
+}
+
+/** Indirect eval → runs the injected script in the scope holding jsdom's document. */
+const evaluate = async (expr: string) => (0, eval)(expr);
```

Biome (2.5.7)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)